### PR TITLE
Upload a page for project

### PR DESCRIPTION
**Pull request by dylanscottlow**
Closes # .

Note this is a co-authored project.

Authors: Dylan Scott Low Y.J. and Stephanie Fu Pei-Yun

### CODE REVIEW

File: content/en/project/duolingo_eeg_ml_classification/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Brainhack-school-project
+
```
Comment (pbellec) [inline content/en/project/duolingo_eeg_ml_classification/index.md:2]:
```suggestion
```

**Review by pbellec — CHANGES_REQUESTED** [review]
Awesome project! 

There's only a small issue in the header for the project to render properly.

I've suggested the changes (removing two lines). 

Please accept the changes and I'll merge the project.

**Comment by dylanscottlow** [comment]
Hi Prof. Bellec,

I've committed the changes you requested. Please let me know if there is anything else needed!

**Comment by pbellec** [comment]
OMG so sorry I missed merging this :( LGTM merging now.

**Review by pbellec — APPROVED** [review]

**Pull request merged**